index.bs

```diff
@@ -2334,6 +2309,121 @@ The [=default allowlist=] for this feature is <code>["self"]</code>.
 
 Note: If the document's origin is not allowed to use the <code>"xr-spatial-tracking"</code> feature policy any [=immersive sessions=] will be blocked, because all [=immersive sessions=] require some use of spatial tracking. [=Inline sessions=] will still be allowed, but restricted to only using the {{XRReferenceSpaceType/"viewer"}} {{XRReferenceSpace}}.
 
+
+Permissions API Integration {#permissions}
```

Comment:
Shouldn't this section have a disclaimer that it's temporary and ought to move to the Permissions API?

Reply:
I'm not sure that's the case? E.g. web bluetooth doesn't have this disclaimer, and it would be really weird to upstream all of this machinery to that API I think.

Reply:
https://w3c.github.io/permissions/#permission-registry is here exactly for that

Reply:
Hmm, okay.